### PR TITLE
PR: Update French translation

### PR DESCRIPTION
**Pull request by goanpeca**
This is a very quick and poor attempt at this. My french is super rusty the computer and software related words are even rustier!

### CODE REVIEW

File: spyder/locale/fr/LC_MESSAGES/spyder.po [hunk 1]
```diff
@@ -50,17 +50,15 @@ msgstr "Fermer le volet courant"
 
 #: spyder/app/mainwindow.py:530
 msgid "Lock panes"
-msgstr ""
+msgstr "Bloques"
```
Comment (SylvainCorlay) [inline spyder/locale/fr/LC_MESSAGES/spyder.po]: Verrouiller les panels?

File: spyder/locale/fr/LC_MESSAGES/spyder.po [hunk 2]
```diff
@@ -692,9 +697,17 @@ msgid ""
 "click on the button below to run some simple code in this console. This will "
 "be useful to show you other important features."
 msgstr ""
+"C'est l'un des volets où vous pouvez exécuter le code que vous avez écrit "
+"sur l'éditeur. Pour ce faire, vous devez appuyer sur la touche <b>F5</b>. "
+"<br><br>Cette console est livrée avec plusieurs fonctionnalités utiles qui "
+"améliorent considérablement votre flux de travail de programmation (comme la "
+"coloration de syntaxe et les graphiques en ligne). Si vous voulez en savoir "
```
Comment (SylvainCorlay) [inline spyder/locale/fr/LC_MESSAGES/spyder.po]: coloration syntaxique

File: spyder/locale/fr/LC_MESSAGES/spyder.po [hunk 3]
```diff
@@ -707,9 +720,14 @@ msgid ""
 "of them, a new window will be opened, where you can inspect and modify their "
 "contents."
 msgstr ""
+"Dans ce volet, vous pouvez visualiser et éditer les variables générées lors "
+"de l'exécution d'un programme, ou celles entrées directement dans une des "
+"consoles Spyder. Comme vous pouvez le voir, l'Explorateur de variables "
+"affiche les variables générées au cours de la dernière etape de cette visite "
```
Comment (SylvainCorlay) [inline spyder/locale/fr/LC_MESSAGES/spyder.po]: étape de la visite (accent)

File: spyder/locale/fr/LC_MESSAGES/spyder.po [hunk 4]
```diff
@@ -971,19 +1003,19 @@ msgstr "Police d'écriture"
 
 #: spyder/plugins/configdialog.py:772
 msgid "Spyder needs to restart to change the following setting:"
-msgstr ""
+msgstr "Il faut redémarrer Spyder pour change l'option suivante"
```
Comment (SylvainCorlay) [inline spyder/locale/fr/LC_MESSAGES/spyder.po]: changer

File: spyder/locale/fr/LC_MESSAGES/spyder.po [hunk 5]
```diff
@@ -971,19 +1003,19 @@ msgstr "Police d'écriture"
...
 
 #: spyder/plugins/configdialog.py:775
 msgid "Spyder needs to restart to change the following settings:"
-msgstr ""
+msgstr "Il faut redémarrer Spyder pour change les options suivantes"
```
Comment (SylvainCorlay) [inline spyder/locale/fr/LC_MESSAGES/spyder.po]: idem

File: spyder/locale/fr/LC_MESSAGES/spyder.po [hunk 6]
```diff
@@ -1054,15 +1083,15 @@ msgstr ""
 
 #: spyder/plugins/configdialog.py:863
 msgid "Enable high DPI scaling"
-msgstr ""
+msgstr "Activer haute PPP mise à l'échelle"
```
Comment (SylvainCorlay) [inline spyder/locale/fr/LC_MESSAGES/spyder.po]: Activer la mise à l'échèle pour les écrans à haute résolution

File: spyder/locale/fr/LC_MESSAGES/spyder.po [hunk 7]
```diff
@@ -3719,14 +3737,12 @@ msgid "Copy path to clipboard"
 msgstr "Copier le chemin d'accès dans le presse-papier"
 
 #: spyder/widgets/editor.py:354
-#, fuzzy
 msgid "Close all to the right"
-msgstr "Fermer tous les fichiers ouverts"
+msgstr "Fermez tous les onglets vers la droite"
```
Comment (SylvainCorlay) [inline spyder/locale/fr/LC_MESSAGES/spyder.po]: I would keep "Fermer"

File: spyder/locale/fr/LC_MESSAGES/spyder.po [hunk 8]
```diff
@@ -4404,19 +4427,20 @@ msgid "Restart kernel?"
 msgstr "Redémarrer le noyau"
 
 #: spyder/widgets/ipythonconsole/client.py:327
-#, fuzzy
 msgid "Error restarting kernel: %s\n"
-msgstr "Une erreur est survenue lors du démarrage du noyau."
+msgstr "Une erreur est survenue lors du démarrage du noyaul: %s\n"
```
Comment (SylvainCorlay) [inline spyder/locale/fr/LC_MESSAGES/spyder.po]: noyau

File: spyder/locale/fr/LC_MESSAGES/spyder.po [hunk 9]
```diff
@@ -4531,54 +4558,49 @@ msgstr ""
...
-#, fuzzy
 msgid "Version control"
-msgstr "Contrôle du débogage"
+msgstr "Gestion de versions"
 
 #: spyder/widgets/projects/configdialog.py:156
-#, fuzzy
 msgid "Use version control"
-msgstr "Contrôle du débogage"
+msgstr "Utilizer gestion de versions"
```
Comment (SylvainCorlay) [inline spyder/locale/fr/LC_MESSAGES/spyder.po]: Utiliser la gestion de version

File: spyder/locale/fr/LC_MESSAGES/spyder.po [hunk 10]
```diff
@@ -4982,6 +4990,8 @@ msgid ""
 "Spyder was unable to retrieve the value of this variable from the console."
 "<br><br>The error mesage was:<br><i>%s</i>"
 msgstr ""
+"Impossible defrécupérer la valeur de cette variable depuis la console. "
```
Comment (SylvainCorlay) [inline spyder/locale/fr/LC_MESSAGES/spyder.po]: de récupérer

**Comment by goanpeca** [comment]
@SylvainCorlay thanks for this :-), sorry my french is very rusty 😢 

Please let me know when you have reviewed most of so I can push the fixes

**Comment by ccordoba12** [comment]
Thanks @SylvainCorlay for your help! Merging now so we can release 3.0.2 tomorrow (at last :-)

**Pull request merged**